src/test/resources/com/puppycrawl/tools/checkstyle/meta/inputs/checks/blocks/RightCurlyCheck.java

```diff
@@ -0,0 +1,232 @@
+ * <a href="https://github.com/checkstyle/checkstyle/issues/5945">#5945</a>.
+ * For right curly brace of enum constant please follow issue
+ * <a href="https://github.com/checkstyle/checkstyle/issues/7519">#7519</a>.
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code option} - Specify the policy on placement of a right curly brace
+ * (<code>'}'</code>).
+ * Type is {@code com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyOption}.
+ * Default value is {@code same}.
+ * </li>
+ * <li>
+ * Property {@code tokens} - tokens to check
+ * Type is {@code java.lang.String[]}.
+ * Validation type is {@code tokenSet}.
+ * Default value is:
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_TRY">
+ * LITERAL_TRY</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_CATCH">
+ * LITERAL_CATCH</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_FINALLY">
+ * LITERAL_FINALLY</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_IF">
+ * LITERAL_IF</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_ELSE">
+ * LITERAL_ELSE</a>.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="RightCurly"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *
+ *   public void test() {
+ *
+ *     if (foo) {
+ *       bar();
+ *     }           // violation, right curly must be in the same line as the 'else' keyword
+ *     else {
+ *       bar();
+ *     }
+ *
+ *     if (foo) {
+ *       bar();
+ *     } else {     // OK
+ *       bar();
+ *     }
+ *
+ *     if (foo) { bar(); } int i = 0; // violation
+ *                   // ^^^ statement is not allowed on same line after curly right brace
+ *
+ *     if (foo) { bar(); }            // OK
+ *     int i = 0;
+ *
+ *     try {
+ *       bar();
+ *     }           // violation, rightCurly must be in the same line as 'catch' keyword
+ *     catch (Exception e) {
+ *       bar();
+ *     }
+ *
+ *     try {
+ *       bar();
+ *     } catch (Exception e) { // OK
+ *       bar();
+ *     }
+ *
+ *   }                         // OK
+ *
+ *   public void testSingleLine() { bar(); } // OK, because singleline is allowed
+ * }
+ * </pre>
+ * <p>
+ * To configure the check with policy {@code alone} for {@code else} and
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">
+ * METHOD_DEF</a> tokens:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RightCurly&quot;&gt;
+ *   &lt;property name=&quot;option&quot; value=&quot;alone&quot;/&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_ELSE, METHOD_DEF&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
```

Comment:
We are not using in meta examples , please make all examples like `public class Test { /** skipped as not relevant */ }`

Reply:
not done.

Reply:
sorry, its done now.